app/packs/src/components/UserLabels.js

```diff
@@ -448,33 +454,24 @@ class ShowUserLabels extends React.Component {
       : [];
 
     if (!MatrixCheck(currentUser && currentUser.matrix, UL_FUNC_NAME)) {
-      return (<span />);
+      return null;
     }
     const elementLabels = (labels || []).filter((r) => (
       (curLableIds || []).includes(r.id) && (r.access_level > 0 || r.user_id === currentUser.id)
     )).map((ll) => (
       <Badge
         key={`bg_${ll.id}`}
+        bg="custom"
         style={{
           backgroundColor: ll.color,
-          color: 'white',
-          borderColor: 'white',
-          borderStyle: 'solid',
-          borderWidth: 'thin',
           borderRadius: (ll.access_level === 1 && ll.user_id !== currentUser.id) ? 'unset' : '10px'
         }}
       >
         {ll.title}
       </Badge>
     ));
 
-    return (
-      <span>
-        &nbsp;
-        {elementLabels}
-        &nbsp;
-      </span>
-    );
+    return ({elementLabels});
```

Comment:
This line throws an error when I start this branch

The return should look like this:
**return (elementLabels);**